### ISSUE TITLE
User tagging issue in comments

### ISSUE THREAD

**Issue by petati94** [PHPfox-Official/phpfox-v4-issues#3023]
# Important

Please follow this template!

----------------------------------------

## What's happened?
...
If my users try tag each other in a comment an mobile (!) with following syntax: @user then following will happen:
@UserUser will be insert when the users push a space after the @User.

## Steps to reproduce:
1. Use Mobile Browser
2. Try to user @User Tagging
3. Type a space rigth after the tag

## What's expected?
No duplicate @UserUser

## Browsers and Devices tested
(Example: Chrome on iPhone X, Safari on Macbook, Miscrosoft Edge on Windown 10, Firefox on Ubuntu 16.04, ...)

Latest Chrome and Latest Firefox
...

## Server information
(Example: CentOS 7, php 7.1 apache)
CloudLinux7, php7.4, Apache
...

## phpFox version
(Example: phpFox 4.6.0)

phpfox 4.8.6
...

## Screenshots
...
This issue was reproduced in the demo site of phpfox (v4.phpfox.com)

**Comment by harrison05** [comment]
Hi @petati94 

Could you provide us more detail about this issue (which device?, screenshot, screen record...) because I checked on Mobile Browser (chrome, safari) both Android and iOS (site v4.phpfox.com) but I can't reproduce this issue.

Regards.

**Comment by petati94** [comment]
Yes, sure. You can find my screen video on this link. https://cloud.petranyidev.hu/s/yzYRNkHCCWSrzgH

I couldn't upload it directly because bigger than 10 MB

Browser: Google Chrome (latest on android)

Thanks in advance.

**Comment by harrison05** [comment]
Hi @petati94 

I saw the issue in your video, but I still can't reproduce it on our demo site and on your site. It's weird
Please see my screen record (Google Chrome on Android).

https://user-images.githubusercontent.com/41947198/128144384-f91154d2-40ac-4939-ae20-f73578a6c17a.MP4

**Comment by petati94** [comment]
Okay. I am totally lost now.
How its possible? 
My phone is Samsung Galaxy S20 with the latest Android 11 build.

Is it possible that this issue is coming with specified keyboard-softwares?

**Comment by petati94** [comment]
Its comfirmed by me.
It is just happened with specific keyboards:
Not working:
* Samsung Keyboard
* Microsoft Swiftkey Keyboard
Working well:
* Anysoftkeyboard

**Comment by alexram1313** [comment]
Reviving this issue as I can replicate this issue also using GBoard.
